docs/content/configuration.md

```diff
@@ -301,5 +301,6 @@ services:
 | --- | --- | --- | --- |
 | `discovery.name` | `string` | Yes | Name of the discovery configuration to download. |
 | `discovery.prefix` | `string` | No (default: `bundles`) | Path prefix to use to download configuration from remote server. |
+| `discovery.decision` | `string` | No (default: `discovery.name`) | Name of the OPA query that will be used to calculate the configuration |
```

Comment:
Nit: it would be good to differentiate between the literal value "discovery.name" and the value of the discovery.name configuration field.

Reply:
I'd replace

```
(default: `discovery.name`)
```

With

```
(default: value of `discovery.name` configuration field)
```